includes/polling/netstats-snmp.inc.php

```diff
@@ -54,7 +54,7 @@
             $value = $data_array[0][$oid];
         }
         else {
-            $value = 'U';
+            $value = null;
```

Comment:
The U is for rrdtools' benefit rather than influx. Think it needs to go back to U.